lib/general.sh

```diff
@@ -1293,7 +1293,11 @@ prepare_host()
 	fi
 
 	if grep -qE "(Microsoft|WSL)" /proc/version; then
-		exit_with_error "Windows subsystem for Linux is not a supported build environment"
+		if [ -f /.dockerenv ]; then
+			display_alert "WSL2 Docker build images may failed" "" "wrn"
```

Comment:
I'd suggest to edit the warning to something like "Building images using Docker on WSL2 may fail"

Reply:
OK